integration/tests/cook/test_client.py

```diff
@@ -41,13 +41,17 @@ def test_query(self):
         job = self.client.query(uuid)
         self.assertEqual(job.command, 'ls')
         self.assertAlmostEqual(job.cpus, 0.5)
-        self.assertAlmostEqual(job.max_retries, 5)
+        self.assertAlmostEqual(job.mem, 1.0)
+        self.assertEqual(job.max_retries, 5)
 
     def test_kill(self):
-        uuid = self.client.submit(command='ls',
+        uuid = self.client.submit(command='sleep 10',
```

Comment:
Just to be on the safe side, let's do:

```suggestion
        uuid = self.client.submit(command=f'sleep {util.DEFAULT_TEST_TIMEOUT_SECS}',
```